src/services/jetstreamService.ts

```diff
@@ -190,9 +192,11 @@ async function createStream(jsm: JetStreamManager, streamName: string, subjectNa
  *
  * @return {*}  {Promise<void>}
  */
-export async function handleResponse(response: string, subject?: string): Promise<void> {
+export async function handleResponse(response: string, subject: string[]): Promise<void> {
   const sc = StringCodec();
-  if (producerStreamName) await js.publish(subject ?? producerStreamName, sc.encode(response));
+  for (const sub of subject) {
```

Comment:
Do a promise.all here instead of awaiting each publish please.

Reply:
Done